autorel/changelog_generator/main/changelog_structures.py

```diff
@@ -11,6 +11,7 @@ class Contributor(object):
     def __init__(self, name, email, url):
         self._name = name
         self._email = email
+        self._url = url
```

Comment:
Done. I did a rebase :sweat_smile: @MrAnno can you please give some quick notes on the class structure changes ?